alibi_detect/cd/mmd.py

```diff
@@ -18,6 +18,7 @@ def __init__(
             x_ref: Union[np.ndarray, list],
             backend: str = 'tensorflow',
             p_val: float = .05,
+            estimator: str = 'quad',
```

Comment:
Would `estimator_complexity` be more descriptive? (Or at least make clear in the docstring)

Reply:
Added extra description in the docstring.